File: win-prog/ProgWinVB/Images and Bitmaps/ProgramWithIcon/ProgramWithIcon.vb
'------------------------------------------------
' ProgramWithIcon.vb  2001 by Charles Petzold
'------------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Class ProgramWithIcon
    Inherits Form

    Shared Sub Main()
        Application.Run(new ProgramWithIcon())
    End Sub

    Sub New()
        Text = "Program with Icon"
        Icon = New Icon(GetType(ProgramWithIcon), "ProgramWithIcon.ico")
    End Sub
End Class

File: win-prog/ProgWinCS/Images and Bitmaps/ProgramWithIcon/ProgramWithIcon.cs
//-------------------------------------------
// ProgramWithIcon © 2001 by Charles Petzold
//-------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class ProgramWithIcon: Form
{
     public static void Main()
     {
          Application.Run(new ProgramWithIcon());
     }
     public ProgramWithIcon()
     {
          Text = "Program with Icon";

          Icon = new Icon(typeof(ProgramWithIcon), 
                          "ProgramWithIcon.ProgramWithIcon.ico");
     }
}